src/pwm.rs

```diff
@@ -1,5 +1,7 @@
 //! Pulse Width Modulation
 
+use core::time::Duration;
```

Comment:
I have not used this so far. How good does it play with `embedded-time`?

Reply:
no idea, but, it avoids an external dependency / seems like it should be viable to have embedded-time play well with this?